Dockerfile.backend

```diff
@@ -0,0 +1,19 @@
+FROM node:14-alpine
+
+WORKDIR /app/
+
+COPY ./package.json ./package.json
+COPY ./packages/backend/config ./packages/backend/config
+COPY ./packages/backend/src ./packages/backend/src
+COPY ./packages/backend/index.js ./packages/backend/index.js
+COPY ./packages/backend/package.json ./packages/backend/package.json
+COPY ./packages/shared/package.json ./packages/shared/package.json
+COPY ./packages/shared/src ./packages/shared/src
```

Comment:
I hope you can make more beauty this piece of code
for the example 
COPY ./package.json .
COPY ./packages/backend/config .
etc